src/storage/cpu_shared_storage_manager.h

```diff
@@ -115,13 +115,18 @@ class CPUSharedStorageManager final : public StorageManager {
 };  // class CPUSharedStorageManager
 
 void CPUSharedStorageManager::Alloc(Storage::Handle* handle) {
+  if (handle->size == 0) {
+    handle->dptr = nullptr;
```

Comment:
The default value of dptr is nullptr, why do we have to set it again? is there some hidden complexity that should be documented?

Reply:
There is no guarantee that the dptr in handle is the default value. Setting it to nullptr in our API makes it sane.